src/helpers/index.js

```diff
@@ -29,3 +31,12 @@ export const ifElse = (predicate, ifSt, elseSt) => {
 };
 
 export const getLocaleItem = (path, locale) => pathOr('', path, locale);
+
+// TODO: find better solution for check browser and getting pathname
+export const isBrowser = new Function('try {return this===window;}catch(e){ return false;}'); // eslint-disable-line
```

Comment:
another googled check for bowsers:
export const isBrowser = typeof window === "object"
    && typeof document === 'object'
    && document.nodeType === 9;

export default isBrowser;